src/openforms/payments/contrib/worldline/plugin.py

```diff
@@ -0,0 +1,255 @@
+from collections.abc import Generator
+from typing import Any
+
+from django.http import (
+    HttpRequest,
+    HttpResponse,
+    HttpResponseBadRequest,
+    HttpResponseRedirect,
+    HttpResponseServerError,
+)
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+import structlog
+from onlinepayments.sdk.api_exception import ApiException
```

Comment:
```suggestion
from onlinepayments.sdk.api_exception import ApiException as WorldlineApiException
```
the name almost collides with `rest_framework.exceptions.APIException` and it confused me, so I think we should make the name more clear